src/application/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM  python:3
+FROM python:3
+
+# Add your existing code here without any modifications
```

Comment:
```suggestion
```

This comment doesn't make any sense at this point, code is added via the `ADD res .`-line. Just leave it out as suggested.

Reply:
Good point. I overlooked this comment